Makefile

```diff
@@ -66,8 +66,8 @@ test:
 
 # This target is used to run Go tests that require docker runtime.
 # Some tests, like pipeline tests, need to have docker available to run.
-test-docker:	
-	go test -cover --tags=docker ./...
+test-docker: build
```

Comment:
We need to add it to the next target `test-fn-render` as well :)

Reply:
done. All add a `test-fn-eval` target.